tests/greendns_test.py

```diff
@@ -10,6 +10,7 @@
 from eventlet.support.greendns import dns
 import tests
 import tests.mock
+from nose.tools import assert_equal
```

Comment:
No nose tools, please. `-d` flag makes up for any convenience built into `assert_equal`.

I can remove it if you're busy. Sorry for delay.

Reply:
> `-d` flag makes up for any convenience built into `assert_equal`

I don't understand this remark, but will introduce a local definition of `assert_equal()` instead of changing all those lines back to `assert A == B`. I do find it valuable for debugging that when they are *not* equal, `assert_equal()` displays the values.

It would be helpful if I could add a comment explaining why we define `assert_equal()` locally rather than using the one from `nose.tools`.

Reply:
```
nosetests --help
...
  -d, --detailed-errors, --failure-detail
                        Add detail to error output by attempting to evaluate
                        failed asserts [NOSE_DETAILED_ERRORS]
```

No nose tools external or internal please.

Reply:
Okay: I now believe you're saying that you *prefer* `assert A == B` because the nose `-d` switch provides the extra information we could have gotten from `assert_equal()`. If so, I completely misread your remark earlier. Will revert source changes accordingly.

Reply:
Yeah and with pytest (some time in future) plain assert is even more helpful pinpointing outliers in data structures.